Реализация скриншотера для нескольких мониторов

skynet2 opened the pull request:
1. Все проекты собираются (интерфейс не трогал, свои модули перенес в другую ветку)
2. Учел пожелания по захвату изображения для нескольких мониторов, пожалуйста, проверьте на соответствие вашим требованиям.

skynet2 committed:
Сделал с опорой на требования

TBXin commented:
Все хорошо, за исключением до сих пор неправильно работающего выделения окна. 
1. Синие рамки, и как следствие захват окон у меня возможен только на 2ом мониторе;
2. Выделение охватывает не все окна;
3. Клик на другом мониторе приводит к захвату экрана второго;

skynet2 committed:
Исправления

skynet2 commented:
1 и 2 баг исправил. 3 не удалось повторить, опишите подробнее.

TBXin commented:
Исправлений не заметил.

Возможно, вы не понимаете как должен работать механизм захвата изображения конкретного окна.

В режиме захвата области экрана пользователь может выбрать произвольную прямоугольную область, однако, есть возможность автоматического выбора области экрана, на которой расположено окно другого приложения. Визуально выбор области окна обозначается синей рамкой без заливки, которая появляется поверх окна, на который пользователь навел курсор мыши.

В текущей реализации синяя рамка появляется сразу и выделяет целиком область второго монитора. При наведении на другие окна синяя рамка не появляется. Единичный клик левой кнопки мыши даже вне этой синей рамки приводит к тому, что захватывается область всего второго экрана (окружена синей рамкой).

skynet2 commented:
Теперь понятно, вечерком гляну.

skynet2 committed:
Click fix

skynet2 committed:
Изменение интерфейса для плагинов
Добавлена загрузка blob обьектов
Изменена загрузка с буфера обмена (поддержка файлов)

skynet2 committed:
Merge remote-tracking branch 'origin/master'

skynet2 committed:
Загрузка с удаленного URL

skynet2 committed:
Azure support

skynet2 committed:
Azure Support 2

skynet2 committed:
Merge pull request #1 from skynet2/private

Private

skynet2 committed:
Windows 10 Fix

skynet2 committed:
no message

skynet2 committed:
Fix,FileUploader

skynet2 committed:
Windows 10 Fix
AzurePlugin

skynet2 committed:
FileUploader Blob Fix

skynet2 committed:
Качество изображений

stas-dm committed:
Win 10 FIx

stas-dm committed:
Buffer support, win10 fixes
